### ISSUE TITLE
Could not find UI helper 'git-credential-manager-ui' with Git 2.39.0

### ISSUE THREAD

**Issue by Safirion** [GitCredentialManager/git-credential-manager#993]
### Version

2.0.877+c7c35983b8

### Operating system

Windows

### OS version or distribution

Windows 10

### Git hosting provider(s)

Other - please describe below

### Other hosting provider

Gitea

### (Azure DevOps only) What format is your remote URL?

None

### Can you access the remote repository directly in the browser?

Yes, I can access the repository

### Expected behavior

Execute git command without error

### Actual behavior

After update Git to the version 2.39.0 on Windows, I can't do anything. Every commands stuck on : 
``warning: could not find UI helper 'git-credential-manager-ui'``

Exemple :
```
$ git pull
warning: could not find UI helper 'git-credential-manager-ui'
```

**Comment by Safirion** [comment]
Actual workaround : downgrade to git 2.38.1

**Comment by dscho** [comment]
I fear that we may lack crucial information, as I cannot reproduce the issue. I also see `C:\Program Files\Git\mingw64\bin\git-credential-manager-ui.exe`, so that should be found on the `PATH`. Maybe it's specific to your provider, in which case the URL is relevant.

**Comment by Safirion** [comment]
How can I help you ?

Because I'm not the only one with this issue :
https://stackoverflow.com/questions/74778511/cant-push-from-git-bash-could-not-find-ui-helper-github-ui

**Comment by abbottbd** [comment]
users in our enterprise are also having this issue.  Using Git 2.38.1 during a clone results in a pop up dialog box asking the users for their credentials.  Using Git 2.39.0 results in an error message:
warning: could not find UI helper 'git-credential-manager-ui'

At the command line a user is then prompted to enter their 'basic' credentials.  If using a 3rd party app then ususally the app will crash and spit out the warning that hte UI helper is missing. 

We are using Azure Devops Server (on prem) to host our repo's.  Please let me know what info I can provide to help get this issue resolved.

**Comment by abbottbd** [comment]
Another work around...manually installing the last released version of GCM from here seems to also resolve the issue: https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.785

**Comment by abbottbd** [comment]
more info - installing latest GCM (2.0.877) on top of latest github also seems to resolve the issue, so perhaps the problem is with how git for windows included the GCM components in their latest install.

**Comment by abbottbd** [comment]
logged on git-for-windows repo also: https://github.com/git-for-windows/git/issues/4165

**Comment by mjcheetham** [comment]
Hey! 👋 Thanks for raising this issue.

The fix for this bug has already been merged in to the `main` branch, and we are in the process of making a new release. Unfortunately we missed the train on getting this GCM release bundled in to the latest Git for Windows.

Please could you try installing the following pre-release to see if this fixes the problem:
https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.886

Thanks!

cc: @dscho

**Comment by rimrul** [comment]
> Please could you try installing the following pre-release to see if this fixes the problem

Seeing as they reported that manually installing 2.0.877 works around the the issue, I'm not sure that's a great test.

**Comment by Safirion** [comment]
> Hey! 👋 Thanks for raising this issue.
> 
> The fix for this bug has already been merged in to the `main` branch, and we are in the process of making a new release. Unfortunately we missed the train on getting this GCM release bundled in to the latest Git for Windows.
> 
> Please could you try installing the following pre-release to see if this fixes the problem: https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.886
> 
> Thanks!
> 
> cc: @dscho

Works fine for me

**Comment by ldennington** [comment]
Closing, as [2.0.886](https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.886) has been promoted to a full release.

**Comment by mbs-c** [comment]
Just as an addendum, as it wasn't mentioned in this issue yet: With Git for Windows 2.39.0, I observed that the text/console fallback for the GitHub UI helper did not appear for everyone. So for some people, git was simply stuck after failing to find the UI helper, while others could simply select the browser login via text prompt.

Is this a separate bug or could that have been caused by the same issue?
